followthemoney/schema/CallForTenders.yaml

```diff
@@ -0,0 +1,77 @@
+CallForTenders:
+  label: "Call for tenders"
+  plural: "Calls for tenders"
+  description: "A public appeal issued by an authority, possibly on behalf of another, for buying a specific work, supply or service"
+  extends:
+    - Thing
+  required:
+    - title
+  featured:
+    - title
+    - contractingAuthority
+  caption:
+    - title
+  properties:
+    title:
+      label: "Title"
+    uid:
+      label: "CfT Unique ID"
+    cftCaUid:
+      label: "CfT CA Unique ID"
+    contractingOnBehalfOf:
+      label: "Published on behalf of"
+    contractingAuthority:
+      label: "Name of Contracting Authority"
+    evaluationMechanism:
+      label: "Evaluation Mechanism"
+    description:
+      label: "Description"
+    procurementType:
+      label: "Procurement Type"
+    directive:
+      label: "Directive"
+    procedure:
+      label: "Procedure"
+    involves:
+      label: "CfT Involves"
+    cpvCodes:
+      label: "CPV Codes"
+    eAuctionsIncluded:
+      label: "Inclusion of e-Auctions"
+    nutsCode:
+      label: "NUTS Code"
+      description: "Nomenclature of Territorial Units for Statistics (NUTS)"
+    aboveOrBelowThreshold:
+      label: "Above or Below threshold"
+    paymentOptions:
+      label: "Payment Options"
+    submissionDeadline:
+      label: "Submission deadline"
+    clarificationDeadline:
+      label: "End of clarification period"
+    clarificationDocumentUpload:
+      label: "Upload of documents within the clarification"
+    openingDate:
+      label: "Tenders Opening Date"
+    notificationAllowed:
+      label: "Allow suppliers to Register for Notifications:"
+    awardedInLots:  
+      label: "Contract awarded in Lots"
+    euFunding:
+      label: "EU funding"
+    multipleTenders:
+      label: "Multiple tenders will be accepted"
+    publicationDate:
+      label: "Date of Publication/Invitation"
+    tedLinksForNotices:
+      label: "TED links for published notices"
+    dateOfAwarding:
+      label: "Date of Awarding"
+    fallsUnderGPPScope:
+      label: "Does this Call fall under the scope of GPP?"
+    certificationCheck:
+      label: "Certification check"
+    tenderers:  
+      label: "Tenderers"
+    awardedContracts:
+      label: "Awarded Contracts"
```

Comment:
We do have an existing `ContractAward` schema, maybe we could just link to that?